packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -120,7 +124,7 @@ export const FileSnapshotWriterClassFactory = <TBase extends ReaderConstructor>(
             this.docId = undefined;
         }
 
-        public onSnapshotHandler(snapshot: IFileSnapshot): void {
+        public onSnapshotHandler(_snapshot: IFileSnapshot): void {
```

Comment:
```suggestion
        public onSnapshotHandler(): void {
```

Reply:
In this case we actually have a site calling the method and passing the argument. It looked as though this was maybe a work in progress, so I erred on the side of leaving things be. But it was only 1 site, so I don't feel strongly one way or the other here.

Reply:
I'd err on the side of nuking unused :-P  We have a lot of odd things lingering around in the repo that don't have an explicit purpose to be there, and we can add it back if needed.